presto-main/src/main/java/com/facebook/presto/metadata/TableLayout.java

```diff
@@ -80,7 +80,7 @@ public TableLayoutHandle getHandle()
                         nodePartitioning.getPartitioningColumns()));
     }
 
-    public Optional<Set<ColumnHandle>> getPartitioningColumns()
```

Comment:
I'd mention, in the commit message, that this should've probably been renamed when stream properties were added and this is just correcting that oversight.

Reply:
I suppose you meant to say "when node properties were added"